db/migrations/20220617130400-remove-duplicates-reposyncstate.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+module.exports = {
```

Comment:
I think it is better to do such things via a temp admin endpoint...

Reply:
I disagree, there's a lot of security considerations of doing SQL changes at the admin level, plus it doesn't really change the end result

Reply:
This is not a migration, but an administrative task. Your change will change the version of the schema, which might cause conflicts if someone else is doing database work, too. (Not saying it is shouldn't be changed.)

Not sure I understand what kind of security problems you think of: an admin endpoint is slauth-protected and can be invoked only by our team.

Given that our migration is currently part of CI, this makes it even worse.

To conclude, I'm disagree with your disagreement, but not blocking you ;)

Reply:
Another point for having this as an admin endpoint: it can be unit-tested, while this migration can't and I only can hope you tested it thoroughly before you will merge/deploy it :)